### PR TITLE
B 19506 validation code feature flag int

### PR DESCRIPTION
**Pull request by cameroncaci**
## Warning
Please approve the [Flipt pull request](https://github.com/transcom/milmove-feature-flags/pull/121) prior to this one

## Summary

Introduces validation code feature flag for MilMove. The Flipt pull request can be found [here](https://github.com/transcom/milmove-feature-flags/pull/121).

### How to test

1. Pull the branch down
2. Run `make feature_flag_docker`
3. Access `http://localhost:9080/` 
4. `validation_code_required` should exist

## Screenshots
image

### CODE REVIEW

File: config/env/prd.app-client-tls.env [hunk 1]
```diff
@@ -31,3 +31,4 @@ FEATURE_FLAG_PPM=false
 FEATURE_FLAG_NTS=false
 FEATURE_FLAG_NTSR=false
 FEATURE_FLAG_CAC_VALIDATED_LOGIN=false
+FEATURE_FLAG_VALIDATION_CODE_REQUIRED=true
```
Comment (danieljordan-caci) [inline config/env/prd.app-client-tls.env:34]: should be false

File: config/env/prd.app.env [hunk 2]
```diff
@@ -39,3 +39,4 @@ FEATURE_FLAG_PPM=false
 FEATURE_FLAG_NTS=false
 FEATURE_FLAG_NTSR=false
 FEATURE_FLAG_CAC_VALIDATED_LOGIN=false
+FEATURE_FLAG_VALIDATION_CODE_REQUIRED=true
```
Comment (danieljordan-caci) [inline config/env/prd.app.env:42]: should be false

File: config/env/prd.app-client-tls.env (hunk 1 above)
Comment (cameroncaci) [reply config/env/prd.app-client-tls.env]: Was under the wrong assumption, resolved! 👍🏻

File: config/env/prd.app.env (hunk 2 above)
Comment (cameroncaci) [reply config/env/prd.app.env]: Was under the wrong assumption, resolved! 👍🏻

**Review by danieljordan-caci — APPROVED** [review]

**Review by traskowskycaci — APPROVED** [review]

**Pull request merged**